README.md

```diff
@@ -286,6 +286,20 @@ This function is mostly used internally, and almost never should be called by yo
 
 jsdom does this for `Window`, which is written in custom, non-webidl2js-generated code, but inherits from `EventTarget`, which is generated by webidl2js.
 
+### For callback interfaces
+
+#### `convert(value, { context })`
+
+Performs the Web IDL conversion algorithm for this callback interface, converting _value_ into a function that performs [call a user object’s operation](https://heycam.github.io/webidl/#call-a-user-objects-operation) when called, with `thisArg` being the `this` value of the converted function.
+
+The resulting function has an `objectReference` property, which is the same object as `value` and can be used to perform identity checks, as `convert` returns a new function object every time.
```

Comment:
Also mention that we don't implement the [callback context](https://heycam.github.io/webidl/#dfn-callback-context) yet.

Reply:
The callback context is used to ensure that the callbacks are executed in the correct realm, which is handled for us by the platform.